bindings/jaxrs/src/main/java/org/commonjava/indy/core/bind/jaxrs/ContentAccessHandler.java

```diff
@@ -380,12 +380,6 @@ public Response doHead( final String packageType, final String type, final Strin
 
                     responseHelper.setInfoHeaders( builder, item, sk, path, true, contentType,
                                     httpMetadata );
-    
-                    if(!path.endsWith("/")) {
-                        // Content hashing headers
-                        builder.header(ApplicationHeader.md5.key(), contentDigester.digest(sk, path, new EventMetadata()).getDigests().get(ContentDigest.MD5));
-                        builder.header(ApplicationHeader.sha1.key(), contentDigester.digest(sk, path, new EventMetadata()).getDigests().get(ContentDigest.SHA_1));
-                    }
```

Comment:
Can we comment it out instead ? Explaining why we need to ignore this.

Reply:
@sswguo sure, done.